app/models/manageiq/providers/ibm_power_hmc/infra_manager/provision/cloning.rb

```diff
@@ -14,9 +14,9 @@ def clone_complete?
   def find_destination_in_vmdb(ems_ref)
     $ibm_power_hmc_log.info("#{self.class}##{__method__}")
     if source.template?
-      ManageIQ::Providers::IbmPowerHmc::InfraManager::Lpar.find_by(:name => dest_name, :ems_ref => ems_ref)
+      source.ext_management_system.vms.find(source.id)
```

Comment:
Hm don't think this is right, we are trying to find the destination by ems_ref (we don't know the dest ID yet)